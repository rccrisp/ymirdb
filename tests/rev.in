set a 1 2 3 4 5
rev a
get a
bye