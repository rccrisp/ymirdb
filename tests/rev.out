> ok

> ok

> [5 4 3 2 1]

> bye
